doc/man/meson.build

```diff
@@ -22,8 +22,13 @@ man5 = [
     'bluechi-agent.conf',
 ]
 
+man8 = [
+    'bluechi-controller.selinux',
+    'bluechi-agent.selinux'
+]
```

Comment:
I forgot to list the old man page files to be deleted in the checklist of the issue.
Please delete them:
- https://github.com/eclipse-bluechi/bluechi/blob/main/selinux/bluechi_agent_selinux.8
- https://github.com/eclipse-bluechi/bluechi/blob/main/selinux/bluechi_controller_selinux.8

Reply:
Deleted previous man pages and their installation